File: example/aa.go
package main

func data() {
}

func main() {
	list := []string{}
	st := "123123"
	for i := range list {
		println(i)
	}
	data();
}

type Data struct {
	Age int
}

func (a *Data) Get(i int, f float32, s string) int  {
	return a.Age
}

File: example/aa.rs
mod rust

enum Shape {
  Circle(int)
  Square(int)
  Rectangle(int, int)
}

trait Comparable {
  fn compareTo()
}

struct Data {
	age: str
}

impl Data {
  fn get(&self, i: int) -> int  {
    return self.age
  }
}

fn main() {
  let data = "123"
  let d = println!("${}")
  for i in 0..1 {
    println(i)
  }
}

fn a(a: i32, b: f32, c: str) 